Sming/Libraries/Rbpf/component.mk

```diff
@@ -1,17 +1,23 @@
+COMPONENT_LIBNAME :=
```

Comment:
With the line above I cannot compile the application. The Rbpf library is not included during the linking. Removing it works as expected.

Reply:
Brain fart. For some reason I thought the library didn't generate any code...